wormchain/app/apptesting/test_suite.go

```diff
@@ -0,0 +1,295 @@
+package apptesting
```

Comment:
Thank you for adding these tests! Can we rename the file to `app_test` or `suite_test` so that go detects it as a test file?

Reply:
If that is done, the file is considered a test file and the `apptesting` package won't include it for `x/tokenfactory/keeper` tests to use. I think it is currently safe as `x/tokenfactory/keeper/keeper_test.go` is the only file that imports that package.